sfaira/commands/templates/single_dataset/{{ cookiecutter.doi_sfaira_repr }}/{{ cookiecutter.id_without_doi }}.yaml

```diff
@@ -8,7 +8,6 @@ dataset_wise:
     doi_preprint:
     doi_journal: "{{ cookiecutter.doi }}"
     download_url_data: "{{ cookiecutter.download_url_data }}"
-    download_url_meta: "{{ cookiecutter.download_url_meta }}"
```

Comment:
We will need this in the template, but I think we can delete it from the questionaire for now.